bot/main.go

```diff
@@ -178,9 +178,12 @@ func createBot(ctx context.Context, flags flags) (*bot.Bot, error) {
 	if err != nil {
 		return nil, trace.Wrap(err)
 	}
-	reviewer, err := review.FromString(flags.reviewers)
-	if err != nil {
-		return nil, trace.Wrap(err)
+	var reviewer *review.Assignments
+	if flags.workflow == "assign" || flags.workflow == "check" {
+		reviewer, err = review.FromString(flags.reviewers)
+		if err != nil {
+			return nil, trace.Wrap(err)
+		}
```

Comment:
I would be very careful about this. This value is used in several other places that are not immediately clear. I took a look into doing the exact same thing a couple of months ago and decided against it because it would break other workflows without much more refactoring.